app/lib/actions/katello/repository_set/enable_repository.rb

```diff
@@ -2,6 +2,8 @@ module Actions
   module Katello
     module RepositorySet
       class EnableRepository < Actions::EntryAction
+        middleware.use Actions::Middleware::BackendServicesCheck
```

Comment:
I wonder, if adding the middleware directly to the backend-service abstract actions, such as https://github.com/jlsherrill/katello/blob/10725/app/lib/actions/pulp/abstract_async_task.rb, wouldn't be better